{"name": "test_add_group[None : namemJ : headerOCgwS 2RycjAL  : footer]", "status": "broken", "statusDetails": {"message": "AttributeError: module pytest has no attribute allure", "trace": "app = <fixture.application.Application object at 0x000001D94CDFD2A0>, db = <fixture.db.DbFixture object at 0x000001D94CDFD480>, json_groups = None : namemJ : headerOCgwS 2RycjAL  : footer, check_ui = False\n\n    def test_add_group(app, db, json_groups, check_ui):\n        group = json_groups\n>       with pytest.allure.step('Given a group list'):\n\ntest\\test_add_group.py:9: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nname = 'allure'\n\n    def __getattr__(name: str) -> object:\n        if name == \"Instance\":\n            # The import emits a deprecation warning.\n            from _pytest.python import Instance\n    \n            return Instance\n>       raise AttributeError(f\"module {__name__} has no attribute {name}\")\nE       AttributeError: module pytest has no attribute allure\n\nenv\\lib\\site-packages\\pytest\\__init__.py:165: AttributeError"}, "parameters": [{"name": "json_groups", "value": "None : namemJ : headerOCgwS 2RycjAL  : footer"}], "start": 1655751686255, "stop": 1655751686255, "uuid": "75f83936-7c16-4b07-97a5-a5fc29e47d6c", "historyId": "9d8535bfcec3692e09a47ed8e897e2fd", "testCaseId": "8e9b1d07ef6cdb67de3c4a3d82e10652", "fullName": "test.test_add_group#test_add_group", "labels": [{"name": "parentSuite", "value": "test"}, {"name": "suite", "value": "test_add_group"}, {"name": "host", "value": "WIN11-Big-PC"}, {"name": "thread", "value": "11336-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test.test_add_group"}]}